File: src/main/java/com/sat/SatApplication.java
package com.sat;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.autoconfigure.security.servlet.UserDetailsServiceAutoConfiguration;
import org.springframework.boot.context.properties.ConfigurationPropertiesScan;

@ConfigurationPropertiesScan
@SpringBootApplication(exclude = {UserDetailsServiceAutoConfiguration.class})
public class SatApplication {

    public static void main(String[] args) {
        SpringApplication.run(SatApplication.class, args);
    }

}

File: src/main/kotlin/com/sat/SatApplication.kt
package com.sat

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.context.properties.ConfigurationPropertiesScan
import org.springframework.boot.runApplication

@ConfigurationPropertiesScan
@SpringBootApplication
class SatApplication

fun main(args: Array<String>) {
	runApplication<SatApplication>(*args)
}
